### ISSUE TITLE
[GOV] Choose and set up password manager for the project

### ISSUE THREAD

**Issue by lmmentel** [aeon-toolkit/aeon#75]

**Comment by aiwalter** [comment]
I got response from NordSec, they want a document that we are non-profit. I guess we cant provide that currently

**Comment by lmmentel** [comment]
We could get a 50% discount from passbolt cloud.

**Comment by MatthewMiddlehurst** [comment]
I don't think we need this anymore IMO with the new governance. It is the responsibility of the workgroup/workgroup leader to secure any resources they use.

**Comment by MatthewMiddlehurst** [comment]
Closing for now. Can discuss if we want this still in the future, but governance has grown in a directory where it is not as necessary IMO.
